src/api/index.spec.tsx

```diff
@@ -359,4 +349,80 @@ describe(__filename, () => {
       expect(getApiHost({ apiHost, useInsecureProxy: false })).toEqual(apiHost);
     });
   });
+
+  describe('makeApiURL', () => {
+    it('constructs an API URL for a given path', () => {
+      const apiHost = 'https://example.org';
+      const path = '/foo/';
+      const _getApiHost = jest.fn().mockReturnValue(apiHost);
+
+      expect(makeApiURL({ _getApiHost, path })).toEqual(
+        `${apiHost}/api/${defaultVersion}${path}`,
+      );
+    });
+
+    it('can omit the version', () => {
+      const path = '/foo/';
+
+      expect(makeApiURL({ path, version: null })).toEqual(
+        expect.stringMatching(`/api${path}`),
+      );
+    });
+
+    it('can omit the prefix', () => {
+      const path = '/foo/';
+      // This is needed because `stringMatching` won't throw if `prefix` is there.
+      const apiHost = 'https://example.org';
+      const _getApiHost = jest.fn().mockReturnValue(apiHost);
+
+      expect(makeApiURL({ _getApiHost, path, prefix: null })).toEqual(
+        `${apiHost}/${defaultVersion}${path}`,
+      );
+    });
+
+    it('can omit both the prefix and version', () => {
+      const path = '/foo/';
+      // This is needed because `stringMatching` won't throw if `prefix` is there.
+      const apiHost = 'https://example.org';
+      const _getApiHost = jest.fn().mockReturnValue(apiHost);
+
+      expect(
+        makeApiURL({ _getApiHost, path, prefix: null, version: null }),
+      ).toEqual(`${apiHost}${path}`);
+    });
+
+    it('can override the default API version', () => {
+      const path = '/foo/';
+      const version = '123';
+
+      expect(makeApiURL({ path, version })).toEqual(
+        expect.stringMatching(`/api/${version}${path}`),
+      );
+    });
+
+    it('can override the default prefix', () => {
+      const path = '/foo/';
+      const prefix = 'another-prefix';
+
+      expect(makeApiURL({ path, prefix })).toEqual(
+        expect.stringMatching(`/${prefix}/${defaultVersion}${path}`),
+      );
+    });
+
+    it('makes sure the path starts with a slash', () => {
+      const path = 'foo';
+
+      expect(makeApiURL({ path })).toEqual(
+        expect.stringMatching(`/api/${defaultVersion}/${path}`),
+      );
+    });
+
+    it('calls _getApiHost', () => {
```

Comment:
This test is already covered but it wouldn't hurt to add `expect(_getApiHost).toHaveBeenCalled()` to the end of it:

https://github.com/mozilla/addons-code-manager/blob/59578a7e340a9a4ab59d62a6c3872e69c28d3c73/src/api/index.spec.tsx#L354-L362